crates/router/src/routes/api_keys.rs

```diff
@@ -166,7 +166,7 @@ pub async fn api_key_update(
     payload.key_id = key_id;
     payload.merchant_id.clone_from(&merchant_id);
 
-    api::server_wrap(
+    Box::pin(api::server_wrap(
```

Comment:
Why is this required?

Reply:
`clippy:large_futures`?

Reply:
Was addressing clippy lints, for `large_futures`